.github/workflows/python.yml

```diff
@@ -140,11 +151,13 @@ jobs:
       # We have a nice script for that: ./scripts/setup_web.sh
       # Unfortunately, we can't run that on Windows, because Windows doesn't come with
       # a package manager like grown-up OSes do (at least not the CI version of Windows).
-      # So we only run the script on !Windows, and then on Windows we do the parts of the script manually.
-      # That way we still get to test the script on Linux and Mac.
+      # Also we can't run it on linux because the 20.04 Docker container will install
+      # an old version of binaryen/wasm-opt that barfs on the `--fast-math` flag
```

Comment:
Hum… we probably shouldn't be using the `--fast-math` flag, but installing the most recent `binaryen/wasm-opt` seems like a good idea anyway